src/constants.js

```diff
@@ -26,3 +26,159 @@ export const T_TETROMINO_LAYERS = [
     { color: 'purple', x: 1, y: 2 }
   ]
 ];
+export const I_TETROMINO_LAYERS = [
```

Comment:
This is forbidden to group all contants in the same file. By the end you will need a `vide-greniers` operation cause it will become UNCONTROLLABLE

To be splitted correctly